### PR TITLE
Update and rename MesonCI.md to MesonCI

### PR DESCRIPTION
**Pull request by frikke**

### CODE REVIEW

File: docs/markdown/MesonCI [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Meson CI setup
+# Meson I setup
```
Comment (xclaesse) [inline docs/markdown/MesonCI:1]: That doesn't make any sense, what are you trying to achieve?

Comment (xclaesse) [reply docs/markdown/MesonCI:1]: Why changing CI to I ?

Comment (eli-schwartz) [reply docs/markdown/MesonCI:1]: And also why delete the correct file extension from a markdown file?

...

This isn't October, but the first thing I thought when I saw this PR was:
https://blog.domenic.me/hacktoberfest/
https://drewdevault.com/2020/10/01/Spamtoberfest.html
https://twitter.com/shitoberfest

**Comment by eli-schwartz** [comment]
This PR is not intended to accomplish anything other than waste maintainers' times rejecting invalid PRs?

**Pull request closed without merging**